src/js/util.js

```diff
@@ -106,10 +106,13 @@ var Util;
          * descendants of the same closest block container. If the preconditions are not met, unexpected
          * behavior will result.
          */
-        createLink: function (document, textNodes, href) {
+        createLink: function (document, textNodes, href, blank) {
```

Comment:
Could you change this argument from `blank` (`boolean`) to `target` (`string`)?  It will be easier to maintain the helper function in the future if you can always pass in the desired target and not just a boolean specifying a specific target.